components/layout/footer/index.tsx

```diff
@@ -20,7 +20,7 @@ const Footer: React.FC = () => {
   const p = footer.pageLinks;
   const pageLinks = [
     [p.projects, "/projekty"],
```

Comment:
Tady by IMO mělo být `/projects`. Ale především by to tady nemělo být hard-coded, měli bysme tu použít `Route.projects`, a tím se opraví i ta chyba. (Podobně odkaz na blog by měl být `Route.blog`.) Opravíš, prosím?